executor/common_linux.h

```diff
@@ -1734,6 +1734,156 @@ static int do_sandbox_namespace(void)
 }
 #endif
 
+#if SYZ_EXECUTOR || SYZ_SANDBOX_ANDROID_UNTRUSTED_APP
+#include <attr/xattr.h>
+
+const uid_t AID_NET_BT_ADMIN = 3001;
+const uid_t AID_NET_BT = 3002;
+const uid_t AID_INET = 3003;
+const uid_t AID_EVERYBODY = 9997;
+const uid_t AID_APP = 10000;
+const char* SELINUX_CONTEXT_UNTRUSTED_APP = "u:r:untrusted_app:s0:c512,c768";
```

Comment:
Most of these unfortunately need to be `#define` because of the aggressive C warnings which are enabled for the repro cases, which do not allow e.g.:

```
const int one = 1;
const int two = a + 1;
```

Since `a + 1` is not a constant expression.

Reply:
I meant the const char* SELINUX_CONTEXT_UNTRUSTED_APP.
It is not const. It is a non-const pointer variable to a const char. To declare a const variable we need: `const char* const`.